tsl/test/sql/bgw_scheduler_restart.sql

```diff
@@ -131,5 +137,8 @@ SELECT (pid != :orig_pid) AS different_pid,
 ALTER SYSTEM RESET timescaledb.bgw_scheduler_restart_time;
 ALTER SYSTEM RESET timescaledb.debug_bgw_scheduler_exit_status;
 SELECT pg_reload_conf();
-
 SELECT _timescaledb_functions.stop_background_workers();
+
+-- We need to restart the launcher as well to read the reset
+-- configuration or it will affect other tests.
+CALL ts_terminate_launcher();
```

Comment:
It says it will restart the launcher, but then "terminate" is called? Is this correct?

Reply:
Yes, the launcher restarts automatically after being terminated.